README.md

```diff
@@ -335,11 +350,11 @@ Might see: `UnhandledPromiseRejectionWarning: browserType.launch: Host system is
 needs to be running locally if you're runnning browser automated tests
 locally.
 
-### Turn on headful mode
+### Turn on headful mode & set Playwright host
 
-So that you can see the browser tests, turn off headless mode. Edit `.env` and set
```

Comment:
Hi @joeltejeda  

Editing the root `.env` file is not read by playwright, so this step was removed, `export` does work

We *might* be able to add a `ln -s` to the `.env` file step instead, I will experiment with this We must avoid duplicate .env files